File: Electricity Bill Estimator.c
#include<stdio.h>
int main() {
  int unit,bill;
  printf("Enter your unit:");
  scanf("%d",&unit);
if(unit<0) {
    printf("invalid unit:");
}else if(unit<=100){
    bill=unit*5;
    printf("your total bill is: %d",bill);
}else if(unit<=200){
    bill=unit*7;
    printf("your Total bill is: %d",bill);
}else{
    bill=unit*10;
    printf("your total bill is: %d",bill);
}
return 0;
}

File: Electricity Bill Estimator.py
unit=int(input("enter your unit"))
if(unit<0):
    print("invalid unit")
elif(unit<=100):
    bill=unit*5
    print("your total bill is:",bill)
elif(unit<=200):
    bill=unit*7
    print("your Total bill is:",bill)
else:
    bill=unit*10
    print("your total bill is:",bill)
